### ISSUE TITLE
Automatically go into "shared" mode when plugged in

### ISSUE THREAD

**Issue by rien333** [baskerville/plato#60]
To make the process of  transferring files as quick as possible, it would be nice to have a setting in the `Settings.toml` file to automatically go into shared mode when the Kobo is plugged into an usb port. The dialog box seems a bit superfluous, given that the only reason that I can think of now for plugging in the device is going into shared mode. Additionally, there really is no way I  will accidentally plug my Kobo into my PC.  

Not sure if you're open to tiny "feature creep" like this, but it fits the minimal character of plato imo. 
Excellent work btw! Can't wait on what the future will hold.
